scripts/services/magazine.service.js

```diff
@@ -1,44 +1,14 @@
-import {
-  getJsonFromUrl,
-  getLanguagePath,
-  getOrigin,
-} from '../common.js';
-
-/**
- * Fetches magazine articles from a given URL.
- * @async
- * @returns {Promise<Array>} - A promise that resolves to an array of article objects or an
- * empty array if the fetch fails.
- */
-export const fetchMagazineArticles = async () => {
-  try {
-    const indexUrl = new URL(`${getLanguagePath()}magazine-articles.json`, getOrigin());
-    const response = await getJsonFromUrl(indexUrl);
-
-    if (!response?.data) {
-      // eslint-disable-next-line no-console
-      console.warn('No data found in response.');
-      return [];
-    }
-
-    const { data } = response;
-
-    return Array.isArray(data) ? data : [data];
-  } catch (error) {
-    // eslint-disable-next-line no-console
-    console.error(`Error fetching articles: ${error.message || error}`);
-    return [];
-  }
-};
+import { getLocale } from '../common.js';
+import { fetchData, magazineSearchQuery, TENANT } from '../search-api.js';
```

Comment:
I know it is not on you but,
What do you think about renaming `fetchData` to `fetchSearchData`?
Seems to generic for me while it is a specific thing...

Reply:
Changed.